pennylane_lightning/src/Apply.hpp

```diff
@@ -55,7 +56,7 @@ namespace Pennylane {
      */
     std::vector<size_t> generateBitPatterns(const std::vector<unsigned int>& qubitIndices, const unsigned int qubits);
 
-    /*
+    /**
```

Comment:
```suggestion
    /*
```

Reply:
Why remove the extra `*`? I just added them since it seems like they should preferably be there for docstrings (that way the syntax highlighting works in my editor as well).

Reply:
Oh, if it's standard then feel free to keep :smile: